ingest/expression_files/mtx.py

```diff
@@ -190,11 +190,21 @@ def transform(self):
             if current_idx != prev_idx:
                 if not MTXIngestor.is_sorted(current_idx, visited_expression_idx):
                     raise ValueError("MTX file must be sorted")
+                GeneExpression.dev_logger.debug(
+                    f"Processing {self.genes[prev_idx - 1]}"
+                )
                 visited_expression_idx.append(current_idx)
                 if prev_idx != 0:
                     # Expressed cells and scores are associated with prior gene
-                    prev_gene_id, prev_gene = self.genes[prev_idx - 1].split("\t")
-                    # Ff the previous gene exists, load its models
+                    features = self.genes[prev_idx - 1].split("\t")
+                    prev_gene_id = features[0]
+                    try:
+                        prev_gene = features[1]
+                    # If no gene_name field is present, hence the index error,
+                    # gene name is equivalent to gene ID
+                    except IndexError:
+                        prev_gene = prev_gene_id
```

Comment:
I think this will be more legible written as
```suggestion
                    if len(features) >= 2:
                        prev_gene = features[1]
                    # If no gene_name field is present,
                    # gene name is equivalent to gene ID
                    else:
                        prev_gene = prev_gene_id
```